### ISSUE TITLE
Accessibility issue due to nested list items in the menu

### ISSUE THREAD

**Issue by clement-escolano** [welpo/tabi#477]
### System information

Zola version: 0.19.2
tabi version: latest

### Expected behaviour

There should be no accessibility issues detected by Lighthouse.

### Actual behaviour

Accessibility score is 92 and Lighthouse warns about `li` items being in a `div` which is inside an `ul` for theme switcher, language switcher and search
https://pagespeed.web.dev/analysis/https-welpo-github-io-tabi/pyyf8jrw0e?form_factor=mobile#accessibility

### Steps to reproduce

Nothing, any zola website with tabi theme would have the issue.
[PageSpeed Insights](https://pagespeed.web.dev) can display the issue as Lighthouse would do

### Final checklist

- [x] I've checked that the issue isn't already reported.
- [x] I've tested with the latest version of tabi to check if the issue has already been fixed.

**Comment by welpo** [comment]
Thanks for reporting this.

I messed up the structure in #425.

Just fixed it in #482. Thanks again!

**Comment by clement-escolano** [comment]
Thank you!
